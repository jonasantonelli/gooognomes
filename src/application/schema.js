export default {
  isError: false,
  isFetching: true
};
