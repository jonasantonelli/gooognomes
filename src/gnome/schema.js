export default {
  isError: false,
  errorMessage: '',
  isFetching: true,
  list: [],
  search: [],
  details: {},
  filter: {
    professions: [],
    minAge: 0,
    maxAge: 100,
    range: {
      min: 20,
      max: 80
    }
  }
};
