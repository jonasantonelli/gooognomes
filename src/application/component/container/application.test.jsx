/**
 * @jest-environment jsdom
 */
import React from 'react';
import thunk from 'redux-thunk';
import { shallow } from 'enzyme';
import configureStore from 'redux-mock-store';
import schema from '../../schema';

import AppContainer from './application.jsx';

const mockStore = configureStore([thunk]);

describe('Application Container', () => {

  let store,
      wrapper;

  beforeEach(() => {
    store = mockStore(schema);
    wrapper = shallow(<AppContainer store={store} />).dive();
  });

  it('match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('renders', () => {
    expect(wrapper.exists()).toBe(true);
  });

});
