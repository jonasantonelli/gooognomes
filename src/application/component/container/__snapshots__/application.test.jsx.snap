// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application Container match snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
